x-pack/test/functional/page_objects/uptime_page.ts

```diff
@@ -51,8 +51,14 @@ export function UptimePageProvider({ getPageObjects, getService }: FtrProviderCo
       await Promise.all(monitorIdsToCheck.map(id => uptimeService.monitorPageLinkExists(id)));
     }
 
-    public async pageUrlContains(value: string) {
-      return await uptimeService.urlContains(value);
+    public async pageUrlContains(value: string, expected: boolean = true, timeout: number = 2000) {
```

Comment:
The common pattern for retires like this are to use `retry.try`. This will try again until the expression does not throw.

```
await retry.try(async () => {
        const isChromeHidden = await PageObjects.common.isChromeHidden();
        expect(isChromeHidden).to.be(true);
      });
```